resources/pithos-initialize.yaml

```diff
@@ -4,12 +4,16 @@ metadata:
   name: pithos-initialize
   namespace: default
 spec:
+  backoffLimit: 10
+  parallelism: 1
```

Comment:
I think we can rid of parallelism and completions since those are the default values:

https://cloud.google.com/kubernetes-engine/docs/how-to/jobs
The default value of completions is 1. When completions is set, the parallelism field defaults to 1 unless set otherwise. If both fields are not set, their default values are 1.

https://kubernetes.io/docs/concepts/workloads/controllers/job/
For a non-parallel Job, you can leave both .spec.completions and .spec.parallelism unset. When both are unset, both are defaulted to 1.